crates/sequencer/src/sequencer.rs

```diff
@@ -813,6 +813,7 @@ where
                             }
                         }
 
+                        // TODO: should not submit commitment if there are skipped blocks
```

Comment:
1. This calls `maybe_submit_commitment`
2. commitment_controller will return an error on missing L1 / L2 connection (btw, this logic is removed in #863 

This comment is not needed.

Reply:
I though that it was a problem because of 2nd point, even though it functioned successfully. Removed https://github.com/chainwayxyz/citrea/pull/861/commits/e8cab68c5ff7b455a6213c074ad6243fa5e17e98